recipes/robotraconteur/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "robotraconteur" %}
+{% set version = "0.18.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/robotraconteur/robotraconteur/archive/refs/tags/v0.18.0.tar.gz
+    sha256: 'f76b6a1a34a9e64279ac1106c4fd6ef19c9048ae1ab2a1331db224f4c3dcd6f6'
+  - url: http://www.kernel.org/pub/linux/bluetooth/bluez-5.56.tar.xz  # [linux]
+    sha256: 59c4dba9fc8aae2a6a5f8f12f19bc1b0c2dc27355c7ca3123eed3fe6bd7d0b9d  # [linux]
+    folder: bluez-src  # [linux]
+
+build:
+  number: 0
+  run_exports:
+  - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
```

Comment:
Do you really want `min_pin='x.x'`?

Reply:
For the moment yes, this will be relaxed soon.